src/writer_kafka.py

```
@@ -3,47 +3,49 @@
+
 def write(topicName, message):
     try:
         _logger.debug(f"Sending to kafka {topicName}")
-        errors = []
+        error = []
         kafka_producer.produce(
             topicName,
             key="",
             value=json.dumps(message).encode("utf-8"),
-            callback=lambda err, msg: errors.append(str(err)) if err is not None else None
+            callback=lambda err, msg: error.append(err) if err is not None else None,
```

Ruff (0.12.2)

Unused lambda argument: `msg`

(ARG005)